client/blocks/login/login-form.jsx

```diff
@@ -140,6 +145,20 @@ export class LoginForm extends Component {
 		} );
 	};
 
+	isJetpackConnectLogin() {
+		const { currentRoute } = this.props;
+		return startsWith( currentRoute, '/log-in/jetpack' );
+	}
+
+	isJetpackCloudLogin() {
+		const { currentRoute, currentQuery } = this.props;
+		return startsWith( currentRoute, '/log-in' ) && currentQuery?.client_id;
```

Comment:
Can we somehow check that `client_id` is indeed a Jetpack cloud client ID? It concerns me a bit that there are other clients (Woo comes to mind) that this code might consider as if they were a Jetpack cloud client id.

Reply:
Yes, very good point!  Ok yes, we can utilize the `getCurrentOAuth2Client()` selector to check that it is a `jetpack-cloud` client_id. 👍